`NewCassandraStore()` can potentially return `nil, nil`

The variable `err` here might be `nil`, so the returned values will be `nil, nil`: https://github.com/grafana/metrictank/blob/master/store/cassandra/cassandra.go#L197

This results in `store.SetTracer()` being called on `nil`, which then errors: https://github.com/grafana/metrictank/blob/master/cmd/metrictank/metrictank.go#L287

```
2018-10-24 14:12:30.003 [INFO] logging level set to 'info'
2018-10-24 14:12:30.014 [INFO] Opened /var/lib/metrictank/partitionOffsets.db
2018-10-24 14:12:30.020 [INFO] kafkamdm: available partitions [0 1 2 3 4 5 6 7]
2018-10-24 14:12:30.021 [INFO] Metrictank starting. Built from 0.10.1-68-g63e4a7e - Go version go1.11.1
2018-10-24 14:12:30.042 [WARNING] cassandra table metric_2048 not found; attempt: 1
2018-10-24 14:12:35.042 [WARNING] cassandra table metric_2048 not found; attempt: 2
2018-10-24 14:12:40.043 [WARNING] cassandra table metric_128 not found; attempt: 3
2018-10-24 14:12:41.090 [INFO] stats now connected to carbon-1-replaytest1:2003
2018-10-24 14:12:45.043 [WARNING] cassandra table metric_16384 not found; attempt: 4
2018-10-24 14:12:50.044 [WARNING] cassandra table metric_512 not found; attempt: 5
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0xd0a3e5]

goroutine 1 [running]:
github.com/grafana/metrictank/store/cassandra.(*CassandraStore).SetTracer(0x0, 0x1085540, 0x1854128)
	/go/src/github.com/grafana/metrictank/store/cassandra/cassandra.go:307 +0x25
main.main()
	/go/src/github.com/grafana/metrictank/cmd/metrictank/metrictank.go:263 +0xc8e
```

--- a/store/cassandra/cassandra.go
+++ b/store/cassandra/cassandra.go
@@ -194,7 +194,7 @@ func NewCassandraStore(config *StoreConfig, ttls []uint32) (*CassandraStore, err
 					if _, ok := keyspaceMetadata.Tables[table.Name]; !ok {
 						log.Warnf("cassandra table %s not found; attempt: %v", table.Name, attempt)
 						if attempt >= 5 {
-							return nil, err
+							return nil, fmt.Errorf("cassandra table %s not found after %d attempts", table.Name, attempt)
 						}
 						time.Sleep(5 * time.Second)
 						continue AttemptLoop

